test/test_helper.rb

```diff
@@ -1,6 +1,3 @@
-require 'coveralls'
-Coveralls.wear!
-
 $:.unshift(File.expand_path(File.dirname(__FILE__) + '../../lib/'))
```

Comment:
I think you are right! I don't know why I took these out, to be perfectly honest.